src/app/content/practiceQuestions/components/Answer/styled.tsx

```diff
@@ -9,6 +9,11 @@ export const AnswerExcerpt = styled.span`
   ${textRegularStyle}
   width: 100%;
   padding: 0;
+  overflow: auto;
+
+  * {
+    overflow: initial;
+  }
```

Comment:
Do we need this for something?

Reply:
It makes the math visible. Similar styles are added to `<ContentExcerpt>` to make the math visible in question content as well.